collectors/ciscoduo/utils.js

```diff
@@ -39,7 +39,8 @@ function getAPILogs(client, objectDetails, state, accumulator, maxPagesPerInvoca
                                 getData();
                             }
                             else {
-                                nextPage = objectDetails.query.mintime;
+                                // Moving time stamp by 1 sec if there is no records;
+                                nextPage = parseInt(objectDetails.query.mintime) + 1;
```

Comment:
I don't quite get why we move it by 1 sec. What is the time interval we poll here for? Should we move move to the end of previous poll interval minus 10 mins?

Reply:
Adding 60 sec to move the window